test/test.js

```diff
@@ -1999,6 +2004,50 @@ describe('Page', function() {
     }));
   });
 
+  describe('Page.select', function() {
+    it('should select single option', SX(async function() {
+      await page.goto(PREFIX + '/input/select.html');
+      await page.select('select', '');
```

Comment:
what does empty string mean as an argument?

Reply:
A non-existent option, could be anything (that's not on the options). Will improve.